overlays/wwinit/init

```diff
@@ -53,4 +53,4 @@ fi
 echo
 echo "There was a problem with the provisioning process, rebooting in 1 minute..."
 sleep 60
-/sbin/reboot
\ No newline at end of file
+echo b > /proc/sysrq-trigger || /sbin/reboot
```

Comment:
Shouldn't it be the other way around? Try to reboot with /sbin/reboot or, if that fails, use sysrq?